src/components/AccountSettings/AdminPanel.tsx

```diff
@@ -123,13 +76,20 @@ class AdminPanel extends Component<Props, State> {
             delete person.username;
           });
         }
-        this.setState({ workers: people });
+        setWorkers(people);
       });
-  }
+  };
+
+  const handleChangeSearchName = (event: any) => {
+    setSearchName(event.target.value);
+    getAdminWorkers(event.target.value);
+  };
 
-  render() {
-    const { currentWorker, adminName, organization, workers } = this.state;
-    const workerPanel =
+  useEffect(() => {
+    getAdminWorkers();
+  }, []);
```

Comment:
does passing `[]` as the second arg mean that the effect only runs once on initial render?